src/views.py

```diff
@@ -100,10 +88,11 @@ def patent_search(patent_title, author=False, status=False):
                                patents=[], error="No patents found")
 
     # Download patent drawings to new folder
-    drawing_files = download_patents.delay(patent_results, destination=dir_name)
+    drawing_files = download_patents.delay(destination=dir_name,
+                                           patent_title=patent_title)
 
     # Extract images from patents' prior art
-    sample_images = get_sample_images.delay(drawing_files, app.static_folder)
+    sample_images = get_sample_images.delay(drawing_files.ready(), app.static_folder)
```

Comment:
not sure `ready()` is the way to go here, the page returned instantly (yes!) but with broken image links (ouch!), probably wanna send an email to the user when celery downloaded the patents?